components/Layout/ContainedLayout.tsx

```diff
@@ -4,17 +4,35 @@ import SeoHead from "../SeoHead"
 import Header from "../Header"
 import Footer from "../Footer/Footer"
 
-function ContainedLayout({ children }: ILayout) {
+function ContainedLayout({ children, hasFooter }: ILayout) {
   return (
-    <div className="min-h-screen w-screen text-black bg-white dark:bg-[black]">
+    <div
+      className="min-h-[100vh] w-screen text-black bg-transparent dark:bg-[black] relative overflow-hidden
+    "
```

Comment:
any reason this `"` is on a new line?
```suggestion
      className="min-h-[100vh] w-screen text-black bg-transparent dark:bg-[black] relative overflow-hidden"
```